sympy/parsing/c/c_parser.py

```diff
@@ -832,10 +913,17 @@ def transform_binary_operator(self, node):
                     combined_variables_stack.append(
                         [token.spelling, 'boolean'])
                 else:
-                    raise NotImplementedError()
+                    raise NotImplementedError(
+                        "Given token {} is not implemented yet!"
+                        .format(token.spelling))
 
             # process remaining operators
             while (len(operators_stack) != 0):
```

Comment:
```suggestion
            while operators_stack:
```

Reply:
Yeah... this seems better, working on it